examples/with-iron-session/.env.production

```diff
@@ -0,0 +1,6 @@
+# ⚠️ The SECRET_COOKIE_PASSWORD should never be inside your repository directly, it's here only to ease
```

Comment:
Can we name this `.env.production.local`? We can exclude the local versions from our repository's `.gitignore` for the examples directory.